p2p/src/main/kotlin/maru/p2p/discovery/MaruDiscoveryService.kt

```diff
@@ -59,15 +60,15 @@ class MaruDiscoveryService(
       .map { NodeRecordFactory.DEFAULT.fromEnr(it) }
       .toList()
 
-  private val discoverySystem: DiscoverySystem =
+  val discoverySystem: MutableDiscoverySystem =
```

Comment:
If you need `nodeRecordBuckets` from `discoverySystem` can you please expose it via `internal` method rather than making `discoverySystem` public?

Reply:
Or even better you can also write the logic to collect all nodes from different buckets into the same collection so it's easier to check in the test

Reply:
That was just for debugging ... Will change it back!